### ISSUE TITLE
arm64 support

### ISSUE THREAD

**Issue by sunjam** [sbrl/Pepperminty-Wiki#188]
I'm attempting to build arm64 compatible version of Pepperminty for use on a Raspberry Pi 4 or Odroid.  Checked the Docker build, but that is for only amd64 so far.

**Comment by sbrl** [comment]
Hey there, @sunjam!

Pepperminty Wiki should run in any environment that PHP runs in. I don't have much experience with Docker myself, I believe that it's @SQL-enwiki that graciously provides the Docker container.

I do have a number of Raspberry Pis of various versions myself, so I can test it there more extensively myself if you like - but I don't yet have a working Docker setup (yet) though.

In short: If it runs PHP, then Pepperminty Wiki should run too :D

**Comment by SQL-enwiki** [comment]
Looking into this now.

**Comment by SQL-enwiki** [comment]
I think I have this figured out.

Are there other arch's that we'd want to support beyond armv71, arm64, and amd64?

**Comment by SQL-enwiki** [comment]
This can be closed. I've got the automated process / buildx up and running, and committing.

**Comment by sbrl** [comment]
Hey, thanks so much @SQL-enwiki!

Erm I think that's pretty much it. The most common ones I can think of are pretty much those, yeah.
